config/samples/meshery_v1alpha1_meshsync.yaml

```diff
@@ -14,4 +14,9 @@ spec:
   broker:
     native:
       name: meshery-broker
-      namespace: meshery
\ No newline at end of file
+      namespace: meshery
+  watch-list:
+    apiVersion: v1
+    data:
+      blacklist: "[\"namespaces.v1.\",\"configmaps.v1.\"]"
+      whitelist: "[{\"Resource\":\"namespaces.v1.\",\"Events\":[\"ADDED\",\"DELETE\"]},{\"Resource\":\"replicasets.v1.apps\",\"Events\":[\"ADDED\",\"DELETE\"]},{\"Resource\":\"pods.v1.\",\"Events\":[\"MODIFIED\"]}]"
```

Comment:
Please follow the same tense for all events. (ADDED, MODIFIED, DELETED OR ADD, MODIFY, DELETE)

Reply:
Let me supply the whole list of possible resources and events as a template that one can then modify as needed